src/Controls/src/Core/MenuItem.cs

```diff
@@ -131,7 +131,7 @@ void IMenuItemController.Activate()
 
 		void OnCommandCanExecuteChanged(object sender, EventArgs eventArgs)
 		{
-			IsEnabledCore = Command.CanExecute(CommandParameter);
+			IsEnabledCore = Command?.CanExecute(CommandParameter) ?? true;
```

Comment:
MAUI's build has `TreatWarningsAsErrors=true`? Do we know why we didn't get a nullable reference type warning/error here?

Might be worth looking into that, to solve more NREs across the whole repo.

Reply:
Nullable is disable on this project Controls.Core

Reply:
Maybe a better fix would be to also turn that on?

Reply:
I think that's because to much breaking changes on "Controls" but i will double check with the rest of the team

Reply:
Seems the idea for the team is to turn it on, and disable per file and go ahead and start turning it on per file so we can do all this work without a mega PR.

Reply:
If I enable it on this file it already gives me errors for all APIs in here because it needs changes in the declared APIs. So this is all breaking changes? Seems like we need to make some kind of project out of this if we want to get this done at some point?

Probably best to do it separate from this fix, if we want to backport, also adding the nullability will prevent us from doing that.

![image](https://user-images.githubusercontent.com/939291/211553259-aa6199ef-1ccb-4499-b3a6-c6a2c354cb6c.png)